.travis.yml

```diff
@@ -8,15 +8,7 @@ language: php
 
 matrix:
   include:
-    - php: 5.5
-    - php: 5.6
-    - php: 7
-    - php: nightly
-    - php: hhvm
-  allow_failures:
-    - php: 5.5
-    - php: nightly
-    - php: hhvm
+    - php: 7.1
```

Comment:
Shouldn't we support `nightly`? But allow failures, with that we can detect issues before a new version is released.